modules/core/include/opencv2/core/persistence.hpp

```diff
@@ -1238,6 +1238,11 @@ inline String::String(const FileNode& fn): cstr_(0), len_(0) { read(fn, *this, *
 
 //! @endcond
 
+
+CV_EXPORTS void cvWriteRawData_Base64(::cv::FileStorage & fs, const void* _data, int len, const char* dt);
```

Comment:
Hello! About changing this API, there is one problem to be discussed.

`cvWriteRawData_Base64` cannot be continually called, like:
```
// in `icvWriteMat`
for( y = 0; y < size.height; y++ )
    cvWriteRawData( fs, mat->data.ptr + (size_t)y*mat->step, size.width, dt );
```

When doing something like this, the output data cannot be merged into a single one. To solve this, I might need to add a private variable to `CvFileStorage` for recording status. Meanwhile, users must call `cvStartWriteStruct(fs, name, CV_NODE_SEQ, "binary")` themselves. It doesn't look good.

Another way, I can change the parameters to `(CvFileStorage * fs, const char* name, const void* _data, int len, const char* dt)` and treat it as a complete data type. But then this function must output all the data at once. Is this ok?